avalanche/__init__.py

```diff
@@ -11,10 +11,10 @@
 
 
 def _avdataset_radd(self, other, *args, **kwargs):
-    from avalanche.benchmarks.utils import AvalancheDataset
+    from avalanche.benchmarks.utils import AvalancheClassificationDataset

```

Comment:
This should be valid for all AvalancheDataset(s), not only for classification datasets.